features/promotion/managing_promotions/promotion_validation.feature

```diff
@@ -71,17 +71,17 @@ Feature: Promotion validation
         Then I should be notified that this value should not be blank
         And promotion with name "Christmas sale" should not be added
 
-    @ui @javascript
+    @api @ui @javascript
     Scenario: Trying to add a new promotion without specifying an item percentage discount
         When I want to create a new promotion
         And I specify its code as "christmas_sale"
         And I name it "Christmas sale"
-        And I add the "Item percentage discount" action configured without a percentage value
+        And I add the "Item percentage discount" action configured without a percentage value for "United States" channel
```

Comment:
I see there's only one channel; why do we need this change?

Reply:
Yes, I could have implemented two solutions. The first one was to take the channel from storage, and the second was to pass the channel to this step. I chose the second solution because it can be used if we have more channels. :stuck_out_tongue: